<?xml version="1.0"?>

<launch>
	<arg name="des_x" default= "-3"/>
	<arg name="des_y" default= "-3"/>
	<param name="des_pos_x" value="$(arg des_x)" />
	<param name="des_pos_y" value="$(arg des_y)" />
	
	<!-- other components -->
	<node pkg="final_assignment" name="wall_follow_service_m" type="wall_follow_service_m.py" required="true" />
	<node pkg="final_assignment" name="go_to_point" type="go_to_point_service_m.py" required="true" />
	
	<!-- services -->
	<node pkg="final_assignment" name="check_position" type="check_position.py" required="true" output="screen" />
	<node pkg="final_assignment" name="points_manager" type="points_manager.py" required="true" output="screen" />
	<node pkg="final_assignment" name="reach_random_pos_service" type="reach_random_pos_service.py" required="true" output="screen" />
	<node pkg="final_assignment" name="reach_user_pos_service" type="reach_user_pos_service.py" required="true" output="screen" />
	<node pkg="final_assignment" name="bug0" type="bug0.py" required="true" />
	
	<!-- user console -->
	<node pkg="final_assignment" name="user_console" type="user_console.py" output="screen" required="true" />
</launch>
